dnf/logging.py

```diff
@@ -209,10 +201,6 @@ def _setup(self, verbose_level, error_level, logfile_level, logdir, log_size, lo
         logger_rpm.addHandler(self.stdout_handler)
         logger_rpm.addHandler(self.stderr_handler)
 
```

Comment:
Why not just paint both log file marks at this point, I don't think it matters much that they will happen a bit later. Then you don't need to forward the levels to `_setup_file_loggers()` and you can also turn them off at this place once, paint both marks and turn them back on.

I would also set the level to `WARNING`, not `SUPERCRITICAL`, so that any errors won't actually be filtered out, but the paint mark at `INFO` will.

Reply:
Great idea! I modified the PR according to your suggestions.